examples/RL/create_gym_example.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+import gym
+import l5kit.environment
+
+
+# TODO: Create a Google colab notebook for environment creation and training.
+
+# Before starting, please download the Lyft L5 Prediction Dataset 2020 and
+# follow the instructions (https://github.com/lyft/l5kit#download-the-datasets)
+# to correctly organise it.
+
+# Set the L5KIT_DATA_FOLDER path and uncomment the line below
```

Comment:
This comment is not correct, it is either "set the environment variable" or "uncomment line below", not both. Setting the env variable and then uncommenting the line below will replace what was set.

Reply:
Okay, just keeping "Uncomment the line below"